libraries/botbuilder-storage-mssql/src/mssqlStorage.ts

```diff
@@ -0,0 +1,105 @@
+import { Storage, StoreItem, StoreItems } from 'botbuilder-core';
+import { config as msconfig, ConnectionPool, NVarChar, IResult, Int } from 'mssql';
+import { MSSQLOptions } from './schema';
+
+/**
+ * @module @botbuildercommunity/storage-mssql
+ */
+
+function getConnectionPool(connection): ConnectionPool {
+    try {
+        return new ConnectionPool(connection);
+    }
+    catch(e) {
+        console.error(e);
+    }
+}
+
+export class MSSQLStorage implements Storage {
+    private options: MSSQLOptions;
+    private connection: msconfig;
+    public constructor(options: MSSQLOptions) {
+        this.options = options;
+        this.connection = {
+            user: options.dbuser,
+            password: options.dbpassword,
+            server: options.dbserver,
+            database: options.db
+        };
+    }
+    public async read(keys: string[]): Promise<StoreItems> { //Should this be an array of numbers?
```

Comment:
Those TODO's will be taken care of before merge?

Reply:
Yeah. They are mostly questions to myself.

Reply:
Keys remain as strings, since the keys aren't explicitly auto-increment values. Removed comment.